scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/infer/predictor/README.md

```diff
@@ -0,0 +1,61 @@
+# Image Classification using Java Predictor
+
+In this example, you will learn how to use Java Inference API to 
+build and run pre-trained Resnet 18 model.
+
+## Contents
+
+1. [Prerequisites](#prerequisites)
+2. [Download artifacts](#download-artifacts)
+3. [Setup datapath and parameters](#setup-datapath-and-parameters)
+4. [Run the image classifier example](#run-the-image-inference-example)
```

Comment:
Why Image Classification in Prediction? Is it possible to use another example even may be a simple Linear Regression model?